reader/views.py

```diff
@@ -2438,9 +2438,12 @@ def _internal_do_post(request, update, cat, uid, **kwargs):
         else:
             return jsonResponse({"error": "Only Sefaria Moderators can add or delete categories."})
 
-        j = request.POST.get("json")
+        if "json" in request.POST:
```

Comment:
Is there any code that still uses the `json` key? If not, let's remove it because we want to be moving to use `request.body` instead of the `json` key.